test/crypto/convert.js

```diff
@@ -65,6 +65,22 @@ describe('convert', () => {
 			const buffer = hexToBuffer(defaultHex);
 			return buffer.should.be.eql(defaultBuffer);
 		});
+		it('should create empty Buffer from non-string', () => {
```

Comment:
Should it? Node 6 and 8 both throw errors if you use a number for example.

Reply:
Will be changed to throw type error if it's not string